README.md

```
@@ -34,26 +30,25 @@ The system is built around a `CoordinatorAgent` that intelligently delegates tas
 
 *   Python 3.8+
 *   Google Cloud SDK installed and configured (for running `LlmAgent` instances live).
-    *   Authenticate using: `gcloud auth application-default login`
+    *   Authenticate using: `gcloud auth login`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)